release-notes/release-notes-galera-26.4.17.txt

```diff
@@ -0,0 +1,39 @@
+# Changes in Galera 4.17 (2024-01-08)
+
+Codership is pleased to announce the release of Galera Replication library 4.17,
+implementing wsrep API version 26.
+
+The library is available as targeted packages and package repositories for a
+number of Linux distributions, including Debian 10 (Buster) and 11 (Bullseye),
+20.04 LTS (Focal) and 22.04 LTS (Jammy), CentOS 7 and RHEL 7, 8, and 9.
+Obtaining packages using a package repository removes the need to download
+individual files and facilitates the deployment and upgrade of Galera nodes.
+
+This and future releases will be available from https://www.galeracluster.com.
+
+Notable changes and fixes in Galera since latest release 4.16:
+- "Zero-level" certification keys are implemented. Currently there are 3 levels
+of resource keys: schema, table, row, yet one common resource used by every
+writeset, a node. The key is added by default to every writeset with the default
+type WSREP_KEY_SHARED. Now, node-wide operations can be performed with the same
+semantics as regular transactions. (codership/galera-bugs#891)
```

Comment:
please don't refer to internal galera-bugs issue tracker, these are not visible to users